SPECS/sos/sos.spec

```
@@ -2,13 +2,14 @@
 Summary:        A set of tools to gather troubleshooting information from a system
 Name:           sos
 Version:        4.1
-Release:        3%{?dist}
+Release:        4%{?dist}
 License:        GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/sosreport/sos
 #Source0:       https://github.com/sosreport/sos/archive/%%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
```

Small request: since we're here, could we also update the spec this way:

```suggestion
Source0:        https://github.com/sosreport/sos/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

---

Discussed offline, I missed the fact that this doesn't work on `1.0-dev`. Please ignore this request and I apologize for the trouble.